exception is thrown while parsing compilation json

[ERROR] [17:04] - Running check failed. list index out of range
Traceback (most recent call last):
  File "/afs/sunrise.ericsson.se/se/app/vbuild/SLED11-x86_64/codechecker/6.2.0/lib/python2.7/libcodechecker/libhandlers/check.py", line 454, in main
    analyze_module.main(analyze_args)
  File "/afs/sunrise.ericsson.se/se/app/vbuild/SLED11-x86_64/codechecker/6.2.0/lib/python2.7/libcodechecker/libhandlers/analyze.py", line 393, in main
    actions += log_parser.parse_log(log_file, parseLogOptions)
  File "/afs/sunrise.ericsson.se/se/app/vbuild/SLED11-x86_64/codechecker/6.2.0/lib/python2.7/libcodechecker/analyze/log_parser.py", line 294, in parse_log
    actions = parse_compile_commands_json(logfile, parseLogOptions)
  File "/afs/sunrise.ericsson.se/se/app/vbuild/SLED11-x86_64/codechecker/6.2.0/lib/python2.7/libcodechecker/analyze/log_parser.py", line 259, in parse_compile_commands_json
    get_compiler_target(parseLogOptions, results.compiler)
  File "/afs/sunrise.ericsson.se/se/app/vbuild/SLED11-x86_64/codechecker/6.2.0/lib/python2.7/libcodechecker/analyze/log_parser.py", line 169, in get_compiler_target
    return parse_compiler_target(err)
  File "/afs/sunrise.ericsson.se/se/app/vbuild/SLED11-x86_64/codechecker/6.2.0/lib/python2.7/libcodechecker/analyze/log_parser.py", line 92, in parse_compiler_target
    if line[0] == target_label:
IndexError: list index out of range

--- a/libcodechecker/analyze/log_parser.py
+++ b/libcodechecker/analyze/log_parser.py
@@ -89,7 +89,7 @@ def parse_compiler_target(lines):
 
     for line in lines.splitlines(True):
         line = line.strip().split()
-        if line[0] == target_label:
+        if len(line) > 1 and line[0] == target_label:
             target = line[1]
 
     return target

